src/storage/kv/store.rs

```diff
@@ -172,13 +175,50 @@ impl Store {
         Ok(())
     }
 
+    /// Compacts the store.
     pub async fn compact(&self) -> Result<()> {
         if let Some(inner) = self.inner.as_ref() {
             inner.compact().await?;
         }
 
         Ok(())
     }
+
+    /// Returns the value associated with the key.
+    pub fn get(&self, key: &[u8]) -> Result<Vec<u8>> {
```

Comment:
Totally agree with you Yusuke, I wanted to add these for non-surrealdb users so that if someone just wants to query the store without having to worry about transaction, they could do so. Also I was confused about how these version APIs would play into the transactional integrity, but given that we are now adding it to read-only transactions, I will remove this

Reply:
Removed!